.github/workflows/test.yml

```diff
@@ -15,6 +15,6 @@ jobs:
 
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.20.4"
```

Comment:
I'd recommend switching to the `go-version-file` ref like we've done here https://github.com/xy-planning-network/second-child/pull/575/files